### ISSUE TITLE
Show post content in an alternative language when translation is not available for the selected language not work

### ISSUE THREAD

**Issue by hacaro76** [qtranslate/qtranslate-xt#971]
Dear all, as per my previous question i think that the "Show post content in an alternative language when translation is not available for the selected language." option seems to not work at all. on my site, i've 2 language (Italian and English). 

Italian is the Default
English is the Alternative

As for options says,  this option could made what i need 

 **Show post content in an alternative language when translation is not available for the selected language.**
_When a page or a post with an untranslated content is viewed, a message with a list of other available languages is displayed, in which languages are ordered as defined by option "Lingua Principale e Ordine". If this option is on, then the content of the first available language will also be shown, instead of the expected language, for the sake of user convenience._

here my settings (i've tried all other order like Italian(default and first and Italian(default and Second). but nothing. 

https://ibb.co/VCYYV6K

all that i need, is that for all language that are different to Italian (by browser language) it switch to engllish. i'm going mad to achieve this, i've made a lot of try but nothing.  hoping that someone can help me.

**Comment by herrvigg** [comment]
The link you sent for the settings is not valid. Be sure you unchecked "Hide posts which content is not available for the selected language". The other option will pick the next language in the list for the content, but the post has first to appear. You should describe the issue better, it seems something is missing in your setup.

**Comment by hacaro76** [comment]
I'm so sorry, my english is not good, and probably i cannot explain better.

here my new settings screenshot:  https://ibb.co/Z2d7339

I've checked Hide posts which content is not available for the selected language".

For make test, and be sure that all works,  every time i clear cookie, cache and then select other language like German, French, or other in Chrome(putting the language  on top of list).

Then i make a refresh.  But unfortunately not works as expected.  I know that this is a github repo and this is not a support page or forum. sorry for this, but at the moment, without your help i'm stucked.

**Comment by herrvigg** [comment]
Don't check the "hide posts" option 😅 
Be sure you enabled the post type enabled in the top of the advanced options.

For the cookies and the language detection it's a different topic, let's keep that in #970.
Here the detected language does not matter much, the alternative languages should be displayed as soon as your current language differs from what is given in the post content.

**Comment by herrvigg** [comment]
In your example, what languages do you have in your post?

**Comment by hacaro76** [comment]
> Don't check the "hide posts" option 😅. (it's unchecked)  https://ibb.co/w651ztP
> Be sure you enabled the post type enabled in the top of the advanced options. 
(all it's ebeabled) https://ibb.co/W3wyQQk
> 
> For the cookies and the language detection it's a different topic, let's keep that in #970.
> Here the detected language does not matter much, the alternative languages should be displayed as soon as your current language differs from what is given in the post content.

$lang  = qtranxf_http_negotiate_language(); (this gave me "it", also if i've set French on Chrome) after clear cookie.

**Comment by herrvigg** [comment]
For the language negotiation let's take this in #970, not here. Let's only write about the post content here.

**Comment by herrvigg** [comment]
Those two problems are completely different issues, if we start to mix up everything it will become impossible to follow the thread.

**Comment by hacaro76** [comment]
> In your example, what languages do you have in your post?

Let me to explain better : 

Case 1 - Default, my language Italian = OK
Case2  - Set English on Chrome = OK works fine and go to English version /en
Case3 - Other Languages (Spanish, French, german etc etc. = NOT OK, because it go always in the Default Italian Site

all that i need, is in the Case 3 it go to English version.

**Comment by hacaro76** [comment]
> Those two problems are completely different issues, if we start to mix up everything it will become impossible to follow the thread.

ok sorry, as you probably understand i'm newbie and i try to reach the solution.

**Comment by herrvigg** [comment]
You're still mixing the two topics and this makes it very hard to follow... Are you using github or just email? See the tickets on github.
- #971 (this one) -> **post content** in alternative language
- #970 -> **language auto-detection**, browser http negotiation

Here, the detected language is not very important. We need first to know in which languages is the post, are you sure the post contains IT + EN?

**Comment by hacaro76** [comment]
> Here, the detected language is not very important. We need first to know in which languages is the post, are you sure the post contains IT + EN?

yes they are

**Comment by hacaro76** [comment]
> > Here, the detected language is not very important. We need first to know in which languages is the post, are you sure the post contains IT + EN?
> 
> yes they are

here my page : https://staging3.oscarcalzature.it/

**Comment by herrvigg** [comment]
So what's the problem with the post content in alternative language? Very hard to understand what you are expecting here 🤔 
Please give some links a clear case illustrating the problem. The alternative language option has nothing to do with the browser detection.

**Comment by hacaro76** [comment]
> So what's the problem with the post content in alternative language? Very hard to understand what you are expecting here 🤔
> Please give some links with a clear case illustrating the problem. The alternative language option has nothing to do with the browser detection.

probably i've explained bad. 
I try to explain with a use case. 

All that i expect is :

if visitors are Italian (bowser language = it)  go to the italian site.  https://staging3.oscarcalzature.it/
if visitors are different to italian (browser language != it). go to the english site.  https://staging3.oscarcalzature.it/en

thats all

**Comment by herrvigg** [comment]
In fact I don't think there's any problem with the post content in alternative language. Let's close this one and move the discussion to #970 for the browser detection.

**Comment by herrvigg** [comment]
> > In your example, what languages do you have in your post?
> 
> Let me to explain better :
> 
> Case 1 - Default, my language Italian = OK
> Case2 - Set English on Chrome = OK works fine and go to English version /en
> Case3 - Other Languages (Spanish, French, german etc etc. = NOT OK, because it go always in the Default Italian Site
> 
> all that i need, is in the Case 3 it go to English version.
> 
> Just to make order here my settings
> 
> General https://ibb.co/w651ztP
> 
> Advanced : https://ibb.co/W3wyQQk



> > So what's the problem with the post content in alternative language? Very hard to understand what you are expecting here 🤔
> > Please give some links with a clear case illustrating the problem. The alternative language option has nothing to do with the browser detection.
> 
> probably i've explained bad.
> I try to explain with a use case.
> 
> All that i expect is :
> 
> if visitors are Italian (bowser language = it) go to the italian site. https://staging3.oscarcalzature.it/
> if visitors are different to italian (browser language != it). go to the english site. https://staging3.oscarcalzature.it/en
> 
> thats all
